python/drned_xmnr/op/filtering/events.py

```diff
@@ -0,0 +1,333 @@
+'''DrNED output events.
+
+Instances of the class `LineOutputEvent` are generated for important
+DrNED output lines.  They are passed to a simple pushdown automaton
+that takes care of generating filtered output and transition events.
+'''
+
+import re
+from .cort import Closeable, coroutine
+
+
+class LineOutputEvent(object):
+    indent = 3 * ' '
+    state_name_regexp = re.compile(r'.*/states/([^/.]*)(?:\.state)?\.(cfg|xml)')
+
+    @staticmethod
+    def indent_line(line):
+        return '{}{}'.format(LineOutputEvent.indent, line)
+
+    def __init__(self, line):
+        self.line = line
+        self.complete = False
+
+    def __str__(self):
+        return 'Line event {} {}'.format(self.__class__.__name__, self.line)
+
+    def mark_complete(self):
+        self.complete = True
+
+    def produce_line(self):
+        return self.line
+
+
+class InitialPrepareEvent(LineOutputEvent):
+    def __init__(self):
+        super(InitialPrepareEvent, self).__init__('Prepare the device')
+
+
+class InitStates(LineOutputEvent):
```

Comment:
I've been looking at this for several weeks. Is there a reason the some event names end in "Event" and others dosen't?

Reply:
I guess I started naming them all "...Event" and eventually became lazy, or something like that.

Reply:
fixed